include/server/domain/game_manager.hpp

```diff
@@ -49,10 +54,19 @@ class game_manager
                 ++it;
             }
         }
+
+        player_mapper_.update( *player );
     }
 
 private:
+    using db_server = db::sqlite_connection;
+    using db_type = db::database<db_server>;
+
     game_state current_state_;
+    db_type db_ { "asciinem" };
+    db::item_mapper<db_type> item_mapper_ { db_ };
+    db::player_mapper<db_type> player_mapper_ { db_ };
+    //    db::backpack_mapper<db_type> backpack_mapper_ { db_ };
```

Comment:
Please consider removing commented out code. That's a bad smell [per Uncle Bob's "Clean Code"] as you should know :-)